ddtrace/llmobs/_llmobs.py

```diff
@@ -176,6 +176,18 @@ def enable(
         atexit.register(cls.disable)
         log.debug("%s enabled", cls.__name__)
 
+    @classmethod
+    def _bedrock_enabled(cls):
+        return "botocore" in ddtrace._monkey._PATCHED_MODULES
+
+    @classmethod
+    def _openai_enabled(cls):
+        return "openai" in ddtrace._monkey._PATCHED_MODULES
+
+    @classmethod
+    def _langchain_enabled(cls):
+        return "langchain" in ddtrace._monkey._PATCHED_MODULES
+
```

Comment:
Can we avoid individual checks here? I imagine as we get more LLMObs integrations this will end up polluting the LLMobs class more than necessary. Can we do something like:
1. Have a dictionary mapping of LLMobs integration names to ddtrace contrib integration names (i.e. `{"bedrock": "botocore", "openai": "openai", ...}`)
2. Use that dictionary mapping to check if `{integration_name} in ddtrace._monkey._PATCHED_MODULES`

Reply:
This would be great if we could actually refactor the `SUPPORTED_INTEGRATIONS` dictionary to just be the text-to-text mapping of LLMObs integration names to contrib integration names, like:
```python
SUPPORTED_INTEGRATIONS = {"bedrock": "botocore", "openai": "openai", "langchain": "langchain"}
```
And use that for both auto patching the integrations in `enable()` and checking if the integration is enabled.
(Note, you can use `**` dictionary unpacking to use as kwargs for `patch(...)`)